### ISSUE TITLE
KV v1 support

### ISSUE THREAD

**Issue by kevkcc** [kyndryl-open-source/hashi-vault-js#49]
**Describe the feature/enhancement**
Can use KV v1 APIs as described here: https://developer.hashicorp.com/vault/api-docs/secret/kv/kv-v1

**Expected behavior**
Currently only KV v2 is supported I think, there are some differences in v2 appends a /data to the secret path: https://developer.hashicorp.com/vault/api-docs/secret/kv/kv-v2#read-secret-version

**Additional context**
Thanks for this library!

**Comment by rod-anami** [comment]
Yes, currently KV V2 is fully supported. I think there's no hurt in adding this previous version of KV.
